### ISSUE TITLE
Add link to split keyboard comparison tool

### ISSUE THREAD

**Issue by jhelvy** [diimdeep/awesome-split-keyboards#22]
Hi,

I built this [tool](https://jhelvy.shinyapps.io/splitkbcompare/) for comparing different split keyboard layouts (source code [here](https://github.com/jhelvy/splitKbCompare)). I figured this would be a good place to include a link to it. I wasn't sure where on the main readme it should go so I'm just mentioning it here.

Cheers,
John

**Comment by kissu** [comment]
Great tool ! 👌🏻

**Comment by eldelacajita** [comment]
That's cool! 

If you added a scaling slider, you could even match the size on screen to the real size, using the scale you added below the images, and a ruler.

**Comment by jhelvy** [comment]
I think that actually wouldn't be that hard to implement, but the "print" button will already print the overlays to a PDF that will print to correct scale.

**Comment by pvinis** [comment]
I think you should definitely add this here. Maybe make a PR? :)

**Comment by jhelvy** [comment]
Just made a PR. I also added a table of contents at the top so that it's easier to jump to the content near the bottom of the readme.

**Comment by abowen** [comment]
Are PR's even merged anymore?
The Lily58 has been there for a long time

**Comment by pvinis** [comment]
Indeed. Maybe @diimdeep could add someone else too as maintainer to help with this?
